table.csv

```diff
@@ -21,8 +21,9 @@ keccak-512,                     multihash,      0x1d,           draft,
 blake3,                         multihash,      0x1e,           draft,     BLAKE3 has a default 32 byte output length. The maximum length is (2^64)-1 bytes.
 sha2-384,                       multihash,      0x20,           permanent, aka SHA-384; as specified by FIPS 180-4.
 dccp,                           multiaddr,      0x21,           draft,
-murmur3-128,                    multihash,      0x22,           draft,
+murmur3-64,                     multihash,      0x22,           draft,
```

Comment:
We should document in the description exactly what we mean by this. What we mean is the version of murmur3 used by UnixFS as that's been the first (and only?) user of this code AFAICT.

Reply:
there isn't a 64-bit version of murmur3 btw, it's usually a truncation of 128 or some other abomination

murmur3 has x86 and x64 versions of 32-bit and 128-bit output lengths so when we invoke "64" we're doing weird things, but that's just the nature of murmurhash unfortunately!

if we need a truncated form because we're using it, then maybe we rename the entry to be clear, `murmur3-x64-128-64` or `murmur3-x64-128-trunc64` ("truncX" is a pattern we've used in the table already but for an entry that zeros out some bits, not actually shrinks the output length proper, but overloading the term should be OK).